src/DmaRequestCore.bsv

```diff
@@ -0,0 +1,99 @@
+import FIFOF::*;
+import PcieTypes::*;
+import DmaTypes::*;
+
+typedef 4096 BUS_BOUNDARY
+typedef 12   BUS_BOUNDARY_WIDTH
```

Comment:
use `TLog#`  to convert BUS_BOUNDARY to BUS_BOUNDARY_WIDTH

Reply:
Future codes will use TLog# to get the width.